src/performance.js

```diff
@@ -0,0 +1,147 @@
+/**
+ * @license
+ * This file is part of the Game Closure SDK.
+ *
+ * The Game Closure SDK is free software: you can redistribute it and/or modify
+ * it under the terms of the Mozilla Public License v. 2.0 as published by Mozilla.
+
+ * The Game Closure SDK is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Mozilla Public License v. 2.0 for more details.
+
+ * You should have received a copy of the Mozilla Public License v. 2.0
+ * along with the Game Closure SDK.  If not, see <http://mozilla.org/MPL/2.0/>.
+ */
+
+ /**
+ * @module performanceMetric
+ *
+ * Measures performance by keeping a window of delta times and providing a score using that window.
+ *
+**/
+
+var engine = null;
+
+var DEFAULT_RANK = 0;
+var DEFAULT_ALLOW_REDUCTION = true;
+var LOWER_BOUND = 10;
```

Comment:
These constant names are not clear, try something like `DEFAULT_FPS_LOWER_BOUND` and `DEFAULT_FPS_UPPER_BOUND`